lib/components/Table/components/filters/DateFilter/DateFilter.tsx

```diff
@@ -50,6 +51,9 @@ function DateFilter<Data>({
     }
   }, [filterValue])
 
+  const fromLabel = timezone ? `FROM (${timezone})` : 'FROM'
+  const toLabel = timezone ? `TO (${timezone})` : 'TO'
```

Comment:
I believe that someday in the future, we'll need to support localization here :)